lib/process-services-cloud/src/lib/task/task-filters/components/edit-task-filters/base-edit-task-filter-cloud.component.ts

```diff
@@ -117,7 +117,9 @@ export abstract class BaseEditTaskFilterCloudComponent<T> implements OnInit, OnC
         label: 'ADF_CLOUD_TASK_FILTERS.STATUS.ALL'
     };
 
+    @Input()
```

Comment:
Are u sure about this? The idea before was that u pass the id and the filter gets fetched by the component calling the API. Instead now you are changing the strategy completely by passing the whole filter as input

Reply:
The problem passing only the `id` is that the preferences API always return the `processInstanceId` `null` and the filter is always empty, passing the taskFilter as input we can detect `processInstanceId` changes and update the filter value

Reply:
and why don't we pass the processInstanceId as input same way the appName is passed or the sortingProperties or the filterProperties but we want to pass the whole filter object?